WordPressKit/HTTPClient.swift

```diff
@@ -7,6 +7,18 @@ struct HTTPAPIResponse<Body> {
     var body: Body
 }
 
+extension HTTPAPIResponse where Body == Data {
+    var bodyText: String? {
+        var encoding: String.Encoding?
+        if let charset = response.textEncodingName {
+            encoding = String.Encoding(ianaCharsetName: charset)
+        }
+
+        let defaultEncoding = String.Encoding.isoLatin1
```

Comment:
What prompted the choice for this locale? 

I did a super quick search and [found this on Wikipedia](https://en.wikipedia.org/wiki/ISO/IEC_8859-1):

> ISO-8859-1 was (according to the standard, at least) the default encoding of documents delivered via [HTTP](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol) with a [MIME type](https://en.wikipedia.org/wiki/Media_type) beginning with "text/" ([HTML5](https://en.wikipedia.org/wiki/HTML5) changed this to [Windows-1252](https://en.wikipedia.org/wiki/Windows-1252)).[[1]](https://en.wikipedia.org/wiki/ISO/IEC_8859-1#cite_note-1)[[2]](https://en.wikipedia.org/wiki/ISO/IEC_8859-1#cite_note-2) As of December 2023, 1.3% of all (and only 14 of the top 1000[[3]](https://en.wikipedia.org/wiki/ISO/IEC_8859-1#cite_note-3)) [web sites](https://en.wikipedia.org/wiki/Website) use ISO/IEC 8859-1.[[4]](https://en.wikipedia.org/wiki/ISO/IEC_8859-1#cite_note-encoding-4)[[5]](https://en.wikipedia.org/wiki/ISO/IEC_8859-1#cite_note-5) It is the most declared single-byte character encoding in the world on the Web, but as Web browsers interpret it as the superset [Windows-1252](https://en.wikipedia.org/wiki/Windows-1252), the documents may include characters from that set.

This make me think that [`.windowsCP1252`](https://developer.apple.com/documentation/swift/string/encoding/windowscp1252) might be another valid choice?

🤷‍♂️

Reply:
I took Alamofire as a reference. 😄

Reply:
👌